pkg/resources/resources_manager_test.go

```diff
@@ -77,7 +77,8 @@ var _ = Describe("ResourcesManger", func() {
                            "deviceIDs": ["1017"],
                            "ifNames": ["ib2", "ib3"]}
            }
-        ]}`
+        ],
+         "periodicUpdateInterval": 60}`
```

Comment:
can you add a test for the default value as well ?

Reply:
added